README.md

```diff
@@ -1,7 +1,35 @@
 # ![NGI-RNAfusion](https://raw.githubusercontent.com/SciLifeLab/NGI-RNAfusion/master/docs/images/NGI-RNAfusion_logo.png)
 
+## *NB* This is work in progress - use at your own risk!
+
 Attempt at setting up a pipeline for fusion detection as an addon to the previously existing NGI-RNAseq pipeline.
 
+## Usage:
+### ` --reads`
+Path to input reads 
+**NB: Must be enclosed in quotes!**
+```
+ --reads 'path/to/data/sample_*_{1,2}.fastq'
+```
+
+### `--star`
```

Comment:
Personally I'd prefer this to be `starfusion` instead - STAR is the name of another tool..